src/coreComponents/physicsSolvers/wavePropagation/sem/acoustic/secondOrderEqn/isotropic/AcousticWaveEquationSEM.cpp

```diff
@@ -790,13 +790,13 @@ real64 AcousticWaveEquationSEM::explicitStepForward( real64 const & time_n,
       if( m_enableLifo )
```

Comment:
There are still two `using namespace` at the beginning of this file:
`using namespace dataRepository;
using namespace fields;`
shouldn't they be removed as well?

Reply:
It is only header files we should avoid. Source files are ok